### ISSUE TITLE
Is it possible to send problem reports without a conventional username+password bugzilla account?

### ISSUE THREAD

**Issue by whereswaldon** [abrt/libreport#706]
I'm a Red Hat Associate, and I'd like to submit bug reports when applications crash to the Red Hat bugzilla instance. However, my bugzilla account uses Red Hat SSO instead of a conventional username + password combination. I haven't been able to figure out how to set a password on my bugzilla account (I suspect it's impossible by design for SSO accounts), so I'm unable to report bugs with libreport.

Is there a known workaround for this? If not, is this a feature request that would be accepted, or is it too niche?

Thanks!

**Comment by mgrabovsky** [comment]
It is only possible to report bugs using a username and password combination at the moment and there are no plans to support other authentication methods.

I can change my Bugzilla password on the “Account Information” settings tab. If you can't see that in your web interface, perhaps the “Forgot Password” function in the login box might be an option.

(For what it's worth, the Bugzilla API also seems to accept API keys as an authentication token, but we would need to see a clear demand to implement this in the reporting plugin.)

**Comment by whereswaldon** [comment]
Thanks for the quick response! I have attempted both changing my password in account settings (fails because I don't know my current password, as the password set for me in the DB is probably random garbage. That's how SSO accounts are normally configured on services like this), and I've tried the forgot password link (asks for my email, then redirects me and fails to send me a reset link).

I think I'm stuck unless a BZ admin helps me reset the password or using API keys is an option, but I totally understand your reservation around adding the API keys without a clearer use-case.
